WebSTR/WebSTR.py

```diff
@@ -238,7 +238,7 @@ def main():
     parser.add_argument("--host", help="Host to run app", type=str, default="0.0.0.0")
     parser.add_argument("--port", help="Port to run app", type=int, default=5000)
     args = parser.parse_args()
-    server.run(debug=False, host=args.host, port=args.port)
+    server.run(debug= os.environ.get("FLASK_DEBUG", 0) == "1", host=args.host, port=args.port)
```

Comment:
maybe just add a comment somewhere that this env variable is not mandatory and it is already safeguarded to default to False if it is not set.